### PR TITLE
fix: Updated the Email template

### LINKED ISSUE #348: Update email template layout
### Feature Description

Our email template currently looks like this:
image

We need to change it to: 
image

Figma:
https://www.figma.com/design/jGpo62JSykdqBAKYQ8iiDg/Classroomio-Web?node-id=6656-21326&t=YiOFwRnQ4fRb0rFv-0

### Use Case

_No response_

### Proposed Solution

_No response_

### Alternatives (optional)

_No response_

### Additional Context

_No response_

### Please check the boxes that apply to this feature request.

- [ ] I have searched the existing feature requests to make sure this is not a duplicate.
- [ ] I have provided a detailed description of the requested feature.
- [ ] I have explained the use case or scenario for this feature.
- [ ] I have included any relevant technical details or design suggestions.
- [ ] I understand that this is a suggestion and that there is no guarantee of implementation.

### PR DESCRIPTION
**Pull request by Chifez**
## What does this PR do?
This PR updates the email template 

Fixes #348 

Screenshot (1129)

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] Chore (refactoring code, technical debt, workflow improvements)
- [x] Enhancement (small improvements)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change adds a new database migration
- [ ] This change requires a documentation update

## How should this be tested?

- run on local host

### Required

- [ ] Filled out the "How to test" section in this PR
- [ ] Self-reviewed my own code
- [ ] Commented on my code in hard-to-understand bits
- [ ] Ran `pnpm build`
- [ ] Checked for warnings, there are none
- [ ] Removed all `console.logs`
- [ ] Merged the latest changes from main onto my branch with `git pull origin main`
- [ ] My changes don't cause any responsiveness issues

### Appreciated

- [ ] If a UI change was made: Added a screen recording or screenshots to this PR
- [ ] Updated the ClassroomIO Docs if changes were necessary

### CODE REVIEW

File: apps/dashboard/src/routes/courses/[id]/lessons/+page.svelte [hunk 1]
```diff
@@ -162,6 +176,11 @@
         </div>
       </Box>
     {:else if $lessons.length}
+      {#if reorder}
+        <p class="text-gray-400 text-center dark:text-white italic text-xs">
+          drag a card to reorder
```
Comment (rotimi-best) [inline apps/dashboard/src/routes/courses/[id]/lessons/+page.svelte:181]: Why is this not in translations?? 😢

File: apps/dashboard/src/routes/courses/[id]/lessons/+page.svelte [hunk 2]
```diff
@@ -138,13 +144,21 @@
...
   <PageNav title={$t('course.navItem.lessons.heading')}>
-    <div slot="widget">
+    <div slot="widget" class="flex gap-2">
       <RoleBasedSecurity allowedRoles={[1, 2]}>
         <PrimaryButton
           label={$t('course.navItem.lessons.add_lesson.button_title')}
           onClick={addLesson}
           isDisabled={!!lessonEditing}
         />
+        {#if isSelfPaced}
```
Comment (rotimi-best) [inline apps/dashboard/src/routes/courses/[id]/lessons/+page.svelte:154]: @Chifez Why is this logic still here?

File: apps/dashboard/src/routes/courses/[id]/lessons/+page.svelte [hunk 3]
```diff
@@ -110,6 +112,9 @@
     );
   }
 
+  function reorderHighlight() {
+    reorder = !reorder;
+  }
```
Comment (rotimi-best) [inline apps/dashboard/src/routes/courses/[id]/lessons/+page.svelte:117]: No need for a function if it just toggles. Add it directly to the onClick event

File: apps/dashboard/src/routes/courses/[id]/lessons/+page.svelte (hunk 1 above)
Comment (rotimi-best) [inline apps/dashboard/src/routes/courses/[id]/lessons/+page.svelte:181]: Why is this not in translations?? 😢

File: apps/dashboard/src/routes/courses/[id]/lessons/+page.svelte (hunk 2 above)
Comment (rotimi-best) [inline apps/dashboard/src/routes/courses/[id]/lessons/+page.svelte:154]: @Chifez Why is this logic still here?

File: apps/dashboard/src/routes/courses/[id]/lessons/+page.svelte [hunk 4]
```diff
@@ -14,7 +14,7 @@
   } from '$lib/components/Course/components/Lesson/store/lessons';
   import PageBody from '$lib/components/PageBody/index.svelte';
```
Comment (rotimi-best) [inline apps/dashboard/src/routes/courses/[id]/lessons/+page.svelte:15]: @Chifez Why is the reorder change with the email template change?

**Review by rotimi-best — CHANGES_REQUESTED** [review]
@Chifez Not sure why the reorder task is mixed with this PR.

File: apps/dashboard/src/routes/courses/[id]/lessons/+page.svelte (hunk 3 above)
Comment (Chifez) [reply apps/dashboard/src/routes/courses/[id]/lessons/+page.svelte:117]: i think i know why,i will delete now and create a new PR

**Comment by Chifez** [comment]
> @Chifez Not sure why the reorder task is mixed with this PR.

I think i know why,i will delete now and create a new PR

**Pull request closed without merging**